documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -165,3 +165,42 @@ spec:
           key: cruise-control-log4j.properties
     # ...
 ----
+
+[[api-security-configuration]]
+[discrete]
+== API security
+
+The Cruise Control API is secured by default to protect the cluster against any potentially destructive Cruise Control operations.
```

Comment:
```suggestion
The Cruise Control REST API is secured by default to protect the cluster against potentially destructive Cruise Control operations.
```

Reply:
Please give an example of such an operation. 

"The Cruise Control REST API is secured by default to protect the cluster against potentially destructive Cruise Control operations, such as decommissioning Kafka brokers."